streamly-coreutils.cabal

```diff
@@ -0,0 +1,33 @@
+cabal-version:       2.4
+-- Initial package description 'streamly-coreutils.cabal' generated by
+-- 'cabal init'.  For further documentation, see
+-- http://haskell.org/cabal/users-guide/
+
+name:                streamly-coreutils
+version:             0.1.0.0
+-- synopsis:
```

Comment:
can you please add a line of synopsis, you can take it from the repo description?